php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -327,6 +328,14 @@ public ClassPropertyDeclarationTree CLASS_VARIABLE_DECLARATION() {
         EOS()));
   }
 
+  public SyntaxToken VISIBILITY_MODIFIER() {
```

Comment:
Why not use this new method in `MEMBER_MODIFIER`?

Reply:
done